src/sass/modules/_m-form-elements.scss

```diff
@@ -75,9 +75,9 @@ button.form-button-disabled {
   // See 18F/web-design-standards#1490
   // This fixes things in the meantime.
   [type="checkbox"] {
-    height: auto !important;
+    height: 1.8rem !important;
```

Comment:
Are we sure this won't mess up layouts on other pages? It looks like this was originally added to fix something with the hca messaging application, but I don't yet know how to fake logging in so I can play with it locally: https://github.com/department-of-veterans-affairs/vets-website/pull/3256/files